### PR TITLE
Feature/CON-12 Add dialogs with related test on item and itemClass deletion

### PR DESCRIPTION
**Pull request by bziondik**
Related to: https://oat-sa.atlassian.net/browse/CON-12
Related to 
https://github.com/oat-sa/tao-core-ui-fe/pull/204
https://github.com/oat-sa/extension-tao-mediamanager/pull/236
https://github.com/oat-sa/tao-core/pull/2721

How to test:
- go to Items page
- try to delete Item
- try to delete Class

Can be tested here http://item-delete_warning.playground.kitchen.it.taocloud.org:49591/

image
image

### CODE REVIEW

**Review by habemuscode — APPROVED** [review]
Maybe we need to consider add another issue for translations in our current supported languages

- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful

**Review by mike-ionut-mihai-sandu-tao — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

File: views/js/controller/items/action.js [hunk 1]
```diff
@@ -18,26 +18,173 @@
...
+            url: url,
+            method: 'POST',
+            data: {
+                uri: uri.decode(actionContext.uri),
+                classUri: actionContext.classUri,
+                id: actionContext.id,
+                signature: actionContext.signature
+            },
+            dataType: 'json'
+        }).then(function (response) {
```
Comment (andrey-ponamarev) [inline views/js/controller/items/action.js:167]: I would move the API calls to the dedicated service/provider

**Review by andrey-ponamarev — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] The feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request's target is not `master`

**Review by adriloma — APPROVED** [review]
Nice feature, good work : )

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] bumping versions are applied (if applicable)

**Pull request merged**